proto/scripts/make_python_roundtrip.py

```diff
@@ -0,0 +1,36 @@
+import os
+import sys
+import json
+import lighthouse_result_pb2 as lhr_pb2
+from proto_preprocessor import preproccess_json_lhr
+from google.protobuf.json_format import Parse, MessageToJson
+
+path = os.path.realpath(__file__)
+path_dir = os.path.dirname(path)
+
+def clean():
+    try:
+        os.remove(path_dir + '/../sample_v2_preprocessed.json')
+    except OSError:
+        pass
+
+# clean workspace
+clean()
+
+# preprocess the sample json
+sample = preproccess_json_lhr(path_dir + '/../../lighthouse-core/test/results/sample_v2.json')
+
+# open json, and convert to proto
+with open(sample, 'r') as f:
+    data = json.load(f)
+
+# make proto message
+lhr = lhr_pb2.LighthouseResult()
```

Comment:
what is this lhr object at this point and the line under it?
it could probably be named more precisely